### DISCUSSION TITLE
Downloadable fonts 404 from npm package

### DISCUSSION THREAD

**Discussion by manstie** [sveltejs/kit#7935] [Q&A] [answered]
I am trying to use [mathlive](https://github.com/arnog/mathlive) inside of sveltekit - I have an example repl here:
https://svelte.dev/repl/6e96160fc8c74b289fc306586b3b113d?version=3.53.1

or if you create a fresh svelte project you can use those same files for a page to reproduce it.
Don't forget to run `npm install --save-dev mathlive`

If you look at the console when you go to the page all of the fonts fail to download.
Running locally it tries to find it in `/node_modules/.vite/deps/fonts/KaTeX_Main-Regular.woff2`, on the repl it tries to find it at `https://svelte.dev/repl/fonts/KaTeX_Main-Regular.woff2`, when deployed with `adapter-node` it's looking for the fonts in `/_app/immutable/chunks/fonts/KaTeX_Main-Regular.woff2`.

TL;DR: fonts don't download.

Bonus problem:

When importing mathlive directly to a svelte file I get a `HTMLElement is not defined` error. Even spicier is that eventually the hack I put in the repl stopped working too.

**Comment by teemingc** [comment, marked as answer]
It seems to work well with a client-side import.
https://svelte.dev/repl/3ac725a80c044845976af9977eb5bf6f?version=3.53.1

Also works if SSR is disabled so maybe that's the issue?
https://stackblitz.com/edit/sveltejs-kit-template-default-3cqhpa?file=src/routes/+page.js

Not sure this should work with SSR (I've never used this package). But you could workaround the flashing with an `await` block and a promise for the `import()` to show a loading skeleton / spinner.

**Reply by manstie** [reply]
That makes sense, I forgot about the SSR stuff.
The answer to the fonts problem I found here: https://github.com/arnog/mathlive/issues/1553#issuecomment-1186546928
Thanks for your response.
